src/styles/blocks/header.scss

```
@@ -0,0 +1,251 @@
+.header {
+    padding-top: 40px;
+
+    @media (max-width: 1000px) {
+        padding-top: 20px;
+    }
+
+    @media (max-width: 768px) {
+        padding-top: 10px;
+    }
+
+    // .header__container
+    &__container {
+    }
+
+    // .header__body
+    &__body {
+        position: relative;
+
+        display: flex;
+        justify-content: space-between;
+        align-items: center;
+    }
+
+    // .header__btn
+    &__btn {
+    }
+
+    // .header__logo
+    &__logo {
+    }
+
+    // .header__menu
+    &__menu {
+        @media (min-width: 992px) {
+            position: absolute;
+            z-index: 1;
+            top: 50%;
+            left: 50%;
+            transform: translate(-50%, -50%);
+        }
+
+        @media (max-width: 768px) {
+            position: fixed;
+            top: -100%;
+            left: 0;
+            z-index: 4;
+
+            width: 100%;
+            height: 100%;
+            padding: 140px 27px 27px;
+
+            background-color: $colorMain;
+            overflow: auto;
+
+            transition-property: top, visibility;
+            transition: 0.3s ease-in-out;
```

You can create veriable for duration
```suggestion
            transition: $duration ease-in-out;
```